### PR TITLE
package clean-up

### PR DESCRIPTION
**Pull request by pawelru**
- removed unnecessary empty first line in examples as this is actually being rendered
- removed `osprey::` prefix in tests as this is redundant inside `osprey`
- fix roxygen warning about documenting package rd file
- fix ggplot lifecycle warnings about deprecated functions

Please review the changes carefully and let me know if there is something you don't like.

### CODE REVIEW

**Comment by m7pr** [comment]
You know @pawelru why we have `ADAE <- osprey::rADAE` in examples instead of `ADAE <- rADAE`. It's becuase `rADAE` exists in a couple more other packages. So we are making sure you use the data directly from this package. For example `{teal.transform}` has this datasets https://github.com/insightsengineering/teal.transform/tree/main/data

File: R/g_butterfly.R [hunk 1]
```diff
@@ -247,28 +247,28 @@ g_butterfly <- function(category,
...
   if (!is.null(block_color)) {
-    pl <- ggplot(NULL, aes_string(x = "y")) +
-      geom_bar(data = counts_r, aes_string(y = "n_i", fill = "bar_color"), stat = "identity") +
-      geom_bar(data = counts_l, aes_string(y = "-n_i", fill = "bar_color"), stat = "identity") +
-      geom_text(data = counts_r, aes_string(y = "label_ypos", label = "n_i"), hjust = 0.9) +
-      geom_text(data = counts_l, aes_string(y = "-label_ypos", label = "n_i"), hjust = -0.9) +
-      geom_text(data = total_text_ann_r, aes_string(y = "label_ypos", label = "n"), fontface = "bold", hjust = -1) +
+    pl <- ggplot(NULL, aes(x = .data$y)) +
+      geom_bar(data = counts_r, aes(y = .data$n_i, fill = .data$bar_color), stat = "identity") +
+      geom_bar(data = counts_l, aes(y = -.data$n_i, fill = .data$bar_color), stat = "identity") +
```
Comment (m7pr) [inline R/g_butterfly.R:252]: what mystery is `-.data` oO or is it a typo?

File: R/g_butterfly.R [hunk 2]
```diff
@@ -247,28 +247,28 @@ g_butterfly <- function(category,
...
+      geom_text(data = counts_r, aes(y = .data$label_ypos, label = .data$n_i), hjust = 0.9) +
+      geom_text(data = counts_l, aes(y = -.data$label_ypos, label = .data$n_i), hjust = -0.9) +
```
Comment (m7pr) [inline R/g_butterfly.R:254]: what mystery is `-.data` oO or is it a typo?

Comment (m7pr) [reply R/g_butterfly.R:254]: aaaah, it's taking the negative value of the column.

File: R/g_butterfly.R (hunk 1 above)
Comment (m7pr) [reply R/g_butterfly.R:252]: aaaah, it's taking the negative value of the column.

**Comment by pawelru** [comment]
> You know @pawelru why we have `ADAE <- osprey::rADAE` in examples instead of `ADAE <- rADAE`. It's becuase `rADAE` exists in a couple more other packages. So we are making sure you use the data directly from this package. For example `{teal.transform}` has this datasets https://github.com/insightsengineering/teal.transform/tree/main/data

True but those are not there - we are in `osprey` and not `teal.osprey` with all of the `teal`-family packages. At least R CMD CHECK does not complain ¯\_(ツ)_/¯

**Comment by m7pr** [comment]
I know, imaginative situation is where you load `osprey` then you load `teal.transform` and then you run examples from `osprey`. Those datasets are exactly the same, but if they ever change, then you'd better have `package::` to ensure which dataset you use.

Also if someone runs `?datasetname` without the `package::` prefix, (s)he may endup with a confusing list of multiple package containing one dataset. If you leave the prefix, the very highly probably someone will do the `?` with a package prefix as well.

Not a big teal. We actually removed prefixes in examples for those datasets in `teal.transform`. Just trying to motivate why it has been there. I'm fine removing it.

**Comment by pawelru** [comment]
> I know, imaginative situation is where you load osprey then you load teal.transform and then you run examples from osprey. Those datasets are exactly the same, but if they ever change, then you'd better have package:: to ensure which dataset you use.

This is actually quite likely - one could be a `teal.osprey` user - therefore with `teal.transform` as well as `osprey`. Alright, I will bring back the prefixes for this. Just because this has been already identified to be an issue. But the problem is more general: the same object name could be exported in multiple packages. I don't think there is a good and robust solution other than bring back prefixes to all (including exported !!) funs.

**Review by donyunardi — APPROVED** [review]
Just one small comment about but everything else looks.
I also tested all of the examples.

**Comment by donyunardi** [comment]
There's one thing that I notice is the function `grob_part()` is being repeated twice in `utils.R`.

https://github.com/insightsengineering/osprey/blob/strict_tests/R/utils.R#L221-L235

https://github.com/insightsengineering/osprey/blob/strict_tests/R/utils.R#L327-L342

The contents are identical.
We should remove one of them?

**Comment by pawelru** [comment]
> One thing I notice is that the function grob_part() is repeated twice in utils.R.

Great catch! I will just simply remove it here. This is meant to be a clean-up.

**Comment by m7pr** [comment]
> This is actually quite likely - one could be a teal.osprey user - therefore with teal.transform as well as osprey. Alright, I will bring back the prefixes for this. 

`teal.osprey` depends on `teal.transform` and `osprey`. I wonder which one gets attached first (teal.transform` or `osprey`) when you attach `teal.osprey`.

**Pull request merged**